analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.cs

```diff
@@ -154,7 +154,24 @@ public void Execute_PersistConstraints()
         [TestMethod]
         public void Execute_PersistSymbols_InsideBlock()
         {
-            var collector = SETestContext.CreateCS("var first = true; var second = false; first = second;", new SetTestConstraintCheck()).Collector;
+            var setter = new PreProcessTestCheck(x =>
```

Comment:
This change should not be needed. `SetTestConstraintCheck` should do the trick. It is its purpose.